tools/lint/rules.py

```diff
@@ -478,3 +478,12 @@ class PromiseRejectsRegexp(Regexp):
     file_extensions = [".html", ".htm", ".js", ".xht", ".xhtml", ".svg"]
     description = "Test-file line has a `promise_rejects(...)` call"
     to_fix = """Replace with promise_rejects_dom or promise_rejects_js or `promise_rejects_exactly`"""
+
+
+class AssertPreconditionRegexp(Regexp):
+    pattern = br"[^.]assert_precondition\("
+    name = "ASSERT-PRECONDITION"
```

Comment:
Use underscores for consistency.

```suggestion
    name = "ASSERT_PRECONDITION"
```

Reply:
I don't care so am happy to change, but fyi rules.py uses hyphens, underscores, and even just spaces for different name entries. There's no consistent separator.

Reply:
Grrh... Never mind then. I only expanded a few lines above and saw `ASSERT_THROWS`.